slurm_ops_manager/slurm_ops.py

```diff
@@ -104,6 +107,17 @@ def configure_slurmctld_hostname(self, slurmctld_hostname):
             slurmctld_hostname
         )
 
+    def slurm_config_nhc_values(self, interval=600, state='ANY,CYCLE') -> dict:
```

Comment:
Do you think we need to expose `slurm_config_nhc_values()`in this way? What are your thoughts on assembling this information in the charm proper instead of having to reach back into slurm_ops_manager to get this?

Reply:
I see why you did it this way now. Possibly this is fine.

Reply:
We can change these values in the charm, see https://github.com/omnivector-solutions/slurm-charms/blob/nhc-installation/charm-slurmd/src/charm.py#L120

I added some default values in there, but they can be fine tuned in the charm code. In _any_ charm that uses `slurm-ops-manager`. For example, lets say we want to add NHC to slurmdbd-charm: we already have everything ready for it, we would  need to 1) enable NHC installation; and 2) add something like `slurm_config.update(self._slurm_manager.slurm_config_nhc_values())` to the code before assembling `slurmdbd.conf`.

I thought of leaving those values configurable in the charm, but with some safe-default values as well.